packages/core/src/browser/dialogs/abstract-dialog.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2017 TypeFox and others.
+ * Copyright (C) 2020 Ericsson and others.
```

Comment:
[minor] please preserve original company `TypeFox`.
If you'd like, you can update the date to `2017-2020` instead although it is not mandatory or enforced.